src/js/views/check-ins/check-in/check-in_views.js

```diff
@@ -65,8 +104,9 @@ const LayoutView = View.extend({
     }));
   },
   showCheckInContent() {
+    const collection = new Backbone.Collection(this.model.get('content'));
```

Comment:
Can we move this to `this.model.getContent()` where the collection is made in the entity?